tests/darwin/torch/dataset_test.py

```diff
@@ -74,8 +74,12 @@ def test_should_correctly_create_a_object_detection_dataset(
         self, team_slug: str, team_extracted_dataset_path: Path
     ) -> None:
         root = team_extracted_dataset_path / team_slug / "coco"
+        print(root)
```

Comment:
Prefer `Logging` lines, but in the torch code, I leave it to the AI team a bit more!

Reply:
Sorry, that is just code that should not be there.

Reply:
Ok, can we remove it?